nservicebus/upgrades/all-versions.include.md

```diff
@@ -709,7 +710,8 @@ _This package has been split into NServiceBus.DataBus.AzureBlobStorage and NServ
 
 | Version   | Released       | Supported until   | Notes                             |
 |:---------:|:--------------:|:-----------------:|:---------------------------------:|
-| 1.5.x     | 2019-08-23     | -                 | -                                 |
+| 1.6.x     | 2019-09-06     | -                 | -                                 |
```

Comment:
why is this changed?

Reply:
I had to run the script to update the supported version and it did update this as well. No idea